alinje/eniac
# Upload excel

Updated database to read CSV file and delete old data.

Tested with a new CSV file: "portNy.csv" where we deleted some data and see if database updated.

Test this:
- run start.js
- run insert.js
- in readCSV.js change path name (row 4) to "portNy.csv"
- run readCSV.js
- in database send query " SELECT * FROM Stocks WHERE name = 'TRYG' "
- if correct, the returned rows should be empty (0 returned rows)
